### ISSUE TITLE
Missing Preview Assets

### ISSUE THREAD

**Issue by danielsaidi** [defagos/SwiftUICollection#1]
Hi! I was really excited to find your blog post about wrapping collection views in SwiftUI, since you have walked the same path that I'm currently investigate, and to find this repo as well!

I have reached the part where I want to focus on the SwiftUI view instead of the cell, so I downloaded this repo to give it a try.

Just a heads up that the example can't build because the preview content assets are missing:

<img width="507" alt="image" src="https://user-images.githubusercontent.com/429927/98548850-4d2c7800-229a-11eb-97df-43cf192fd126.png">

You just have to clear that row for the app to build.

And what an app! It runs great on device and I just love how you can provide varying heights for various sections. Amazing!

**Comment by defagos** [comment]
Hi! Glad if this article can help you.

You are right, I had a local empty folder (ignored by git), so I didn't see the demo project was not compiling properly. I just pushed a commit where the settings has been cleared, as you suggested.

Thanks for reporting this issue!

**Comment by danielsaidi** [comment]
It more than helped me, it literally saved me! I came as far as to the focus problem myself, which luckily led me to Google with a term that took me to your article. Besides solving the focus, you also use diffable datasources, which I haven't used before, so my implementation would have been subpar to yours even if I had solved the focus problem.

If you setup a GitHub sponsor, I'd be happy to setup a monthly donation!
